services/docker-compose.yml

```diff
@@ -6,6 +6,7 @@ services:
     hostname: "{{.Node.Hostname}}-{{.Service.Name}}-{{.Task.Slot}}"
     environment:
       - API_SERVER_DEV_FEATURES_ENABLED=${API_SERVER_DEV_FEATURES_ENABLED}
+      - API_SERVER_DEV_HTTP_CALLS_LOGS_PATH=${API_SERVER_DEV_HTTP_CALLS_LOGS_PATH}
```

Comment:
should .env-devel not be updated too?

Reply:
In fact it is already here: https://github.com/bisgaard-itis/osparc-simcore/blob/83d45a697d22566978e6aedac6a83888f803fb8f/.env-devel#L252
It is just commented out.

Reply:
This env var is only used for development purposes and setting it to some value enables the tool. Hence, I prefer if we do not set it directly in the deployment env vars.